Source/items.cpp

```diff
@@ -3709,7 +3709,7 @@ bool DoOil(Player &player, int cii)
 	case IPL_PERIL:
 		return _("2x dmg to monst, 1x to you");
 	case IPL_JESTERS:
-		return std::string(_(/*xgettext:no-c-format*/ "Random 0 - 500% damage"));
+		return std::string(_(/*xgettext:no-c-format*/ "Random 0 - 600% damage"));

```

Comment:
Can you also update the translations?

Reply:
Is there any reason why the numbers are not parameterized in the localization? Having to update translations because of a numerical change like this seems sub-optimal.

Reply:
iirc this was the only affix with a hardcoded number - we don't really expect it to change ;)

Reply:
> Can you also update the translations?

done

Reply:
> iirc this was the only affix with a hardcoded number - we don't really expect it to change ;)

Even if that's the case, it is still beneficial to parameterize it for translation purposes IMHO.